shared/js/textChatStore.js

```diff
@@ -148,13 +155,24 @@ loop.store.TextChatStore = (function() {
     },
 
     /**
-     * Handles sending of a chat message.
+     * Handles sending of a chat message.  Adds a name (XXX yuck, this
+     * shouldn't be so side-effecty)
```

Comment:
I was overthinking it.  You're right that the code is fine as is.  I've removed the comment.  I've also added comments and semantic changes to make sure things are ok if the displayName hasn't yet been set.